exercises/forth/example.go

```diff
@@ -10,10 +10,10 @@ import (
 
 type operatorFn func(stack *[]int) error
 
-type operatorId byte
+type operatorID byte
 
 const (
-	addOp operatorId = iota
```

Comment:
I think they should be `opAdd`, `opSub`, etc. similar to `http.StatusXxx`. However, they aren't exported so it might be fine. What do you think ?